Modules/Core/Support/ClientHelper.php

```
@@ -0,0 +1,64 @@
+class ClientHelper
+{
+    /**
+     * @param obj|int $client     (or id - since 1.6.3)
+     * @param bool    $show_title - since 1.6.3
+     */
+    public static function format_client($client, $show_title = true): string
+    {
+        // Get an id
+        if ($client && is_numeric($client)) {
+            // TODO: Migrate remaining CodeIgniter dependencies to Laravel
+            if ( ! property_exists($CI, 'mdl_clients')) {
```

PHPMD (2.15.0)

Avoid using undefined variables such as '$CI' which will lead to PHP notices. (undefined)

(UndefinedVariable)